Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/README.md

```diff
@@ -0,0 +1,1703 @@
+## Overview
+---
+
+VMware Carbon Black Enterprise EDR is an advanced threat hunting and incident response solution delivering continuous visibility for top security operations centers (SOCs) and incident response (IR) teams. (formerly known as ThreatHunter)
```

Comment:
do we have a version to mention here or is it saas?

Reply:
It is saas, no version